verinfast/cloud/aws/instances.py

```diff
@@ -0,0 +1,76 @@
+import json
+import os
+
+import boto3
+
+import regions as r
```

Comment:
On my machine, this needs to be:
`import cloud.aws.regions as r`
Only the parent script that is run has its path in the sys path.